android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/FileExtensions.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.engine.util.extension
+
+import android.util.Base64
+import android.util.Base64OutputStream
+import java.io.ByteArrayOutputStream
+import java.io.File
+import java.io.FileInputStream
+
+fun File.encodeToBase64(): String {
+  val byteArrayOutputStream = ByteArrayOutputStream()
+  val base64FilterStream = Base64OutputStream(byteArrayOutputStream, Base64.DEFAULT)
+  var fileInputStream: FileInputStream? = null
+  try {
```

Comment:
Use kotlin [use](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/use.html) function for the `ByteArrayOutputStream` and `FileInputStream` classes implementing [Closeable](https://docs.oracle.com/javase/8/docs/api/java/io/Closeable.html)

Reply:
Okay. That greatly reduced the lines of code to 5